### ISSUE TITLE
The LJH files in src/mass/regression_test/ seem to be wrong

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#79]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/79) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Galen, where did you get the files `src/mass/regression_test/regress_chan1.*` ? Something looks truly F'ed about them. I think they have 18-byte headers on each pulse instead of 16-byte headers. The expected header length for LJH 2.2.0 is 16 bytes per pulse. When I run `test_regression.py`, I get the following report:

```
#!text

686massbook2:mass fowlerj$ python src/mass/regression_test/test_regression.py
postpeak_deriv=(0, 3.5) is assigned
peak_value=(0.0, 17000.0) is assigned
rise_time_ms=(0.1, 0.45) is assigned
pretrigger_rms=(None, 5.5) is assigned
pretrigger_mean_departure_from_median=(-50, 50) is assigned
peak_time_ms=(0.2, 0.7) is assigned
pulse_average=(0.0, 1700.0) is assigned
Warning: The binary size (572959) is not an integer multiple of the pulse size 3016 bytes
/Users/fowlerj/Software/mass/src/mass/regression_test/regress_chan1.ljh
/Users/fowlerj/Software/mass/src/mass/regression_test/regress_chan1.ljh 189
Warning: The binary size (601801) is not an integer multiple of the pulse size 3016 bytes
/Users/fowlerj/Software/mass/src/mass/regression_test/regress_chan1.noi
channel.summarize_data_tdm chan 1 100.0% done, estimated ? left
channel.summarize_data_tdm chan 1 finished in 0.0 min
summarize_data 100.0% done, estimated ? left
summarize_data finished in 0.0 min
1 1
Chan 1 after cuts, 0 are good, 189 are bad of 189 total pulses
/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/mass-0.5.0-py2.7-macosx-10.11-x86_64.egg/mass/mathstat/power_spectrum.py:140: RuntimeWarning: invalid value encountered in divide
  return self.specsum / self.nsegments
/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/mass-0.5.0-py2.7-macosx-10.11-x86_64.egg/mass/core/channel.py:329: RuntimeWarning: invalid value encountered in divide
  ac /= entries
chan 1 flagged bad because No good pulses
V/dV for time, Fourier filters: 
...
----------------------------------------------------------------------
Ran 3 tests in 0.046s

OK

```

Notice `Warning: The binary size (572959) is not an integer multiple of the pulse size 3016`, which by the way is already a clue: you have 1500-sample pluses, thus 3000 bytes of data in the pulse record. If the warning says that pulse sizes are 3016, there you go: headers are 16 bytes/pulse. Also, you can just see that there's something wrong this way:

```
#!bash

od -Ad -tx2 -j1197 src/mass/regression_test/regress_chan1.ljh | less
```
...because the header takes up one line (16 bytes) plus two bytes of the next line.

Notice also that all 189 pulses are cut.

I am trying to work on a fix for issue #75 (make V/dV predictions work for new-style filters), and I cannot use these example data because they are totally corrupted. Is this something that @danbek caused, or you?

**Comment by joefowler** [comment]
**Original comment by Galen O'Neil (Bitbucket: [oneilg](https://bitbucket.org/oneilg), GitHub: [oneilg](https://github.com/oneilg)).**

----------------------------------------

I think @danbek added them. I just ran the tests that were there.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

p.s. Yes, I did read the `readme.md` file. At the bottom, it implies that you generated the LJH test files by a certain command that I don't recognize at all. If that's correct, then you should file another issue or add it to this one, because this discovery shows that *something* about how they were generated is buggy.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Dan, I'm reassigning to you, but of course it's really up to you and me both. Tell me (in more detail) how you made the files, and then we'll figure out how to make them correctly...and add a regression test for that!

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Regression test LJH data files were corrupted. Fixes #79.

Had to change cuts. Now using some APS data at 600 eV for regression tests.
